### ISSUE TITLE
OpenCollective URLs returning 403 Forbidden in lychee link checker

### ISSUE THREAD

**Issue by smita1078** [checkstyle/checkstyle#21371]
The lychee link checker is reporting 5 errors for OpenCollective URLs with 403 Forbidden status:

```
https://opencollective.com/checkstyle/ (403)
https://opencollective.com/checkstyle/backers/badge.svg (403)
https://opencollective.com/checkstyle/sponsors/badge.svg (403)
https://opencollective.com/checkstyle/tiers/backer.svg?avatarHeight=76&width=600 (403)
```

These URLs appear in target/site/sponsoring.html
OpenCollective appears to be blocking automated link checker requests
